src/Login/Login.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { ChangeEventHandler, Component } from 'react';
+
+type State = {
+  email: '';
+  password: '';
+};
+
+export class Login extends Component {
+  public state: State = {
+    email: '',
+    password: '',
+  };
+
+  public render() {
+    return (
+      <div>
```

Comment:
No styling of any sort?

Reply:
I was going to get all of the functionality in and working before starting on any styling.

Realistically I'll style once I have react router in place and all my pages nicely separated.